src/services/CertificateGenerationService.ts

```diff
@@ -72,21 +72,18 @@ class CertificateGenerationService {
     const config: IMOTConfig = this.config.getMOTConfig();
     const iConfig: IInvokeConfig = this.config.getInvokeConfig();
     const testType: any = testResult.testTypes;
-    const { STOP_WELSH_GEN } = process.env;
 
-    let isTestStationWelsh = false;
+    const flags = await getProfile();
```

Comment:
such a minor point, can we call this featureFlags, think `flags` is just too general